schema/anyof_test.go

```diff
@@ -126,7 +132,7 @@ func TestAnyOfQueryValidate(t *testing.T) {
 			Name:      `{Bool,Bool}.Validate("")`,
 			Validator: schema.AnyOf{&schema.Bool{}, &schema.Bool{}},
```

Comment:
First of all, no it doesn't make sense in real life, secondly, I don't think it should error during compilation; that would just lead to unpredictable semantics, not to mention it would be hard to implement. Repeating the same `FieldValidator` _type_ with different parameters certainly is valid. 

If we compare it to JSON Schema, it does not really make sense to repeat the same schema in an anyOf clause, but it's still 100% valid according to the spec.

For the sake of this test, I think it's fine as it is.